### ISSUE TITLE
Sklearn get_feature_names() deprecated

### ISSUE THREAD

**Issue by Tebirkesueth** [jbms/beancount-import#203]
Apologies if this is wrongly posted as an issue - I'm rather new to github. 

After reinstalling my virtual environment to clean up my project I started running into issues with the classifer part of the package. At first I thought my old transactions file had some error in it so I decided to try and create a new one for testing purposes. I kept running into the same error ("AttributeError: 'CountVectorizer' object has no attribute get_feature_names"). 

After some searching I found a uni-course that had a similar issue, which concluded that the get_feature_names was marked as deprecated after version 1.0 of sklearn. 

I tried replacing, 
feature_names = classifier._vectorizer.get_feature_names() on line 139 in reconclie.py 
with,
feature_names = classifier._vectorizer.get_feature_names_out(), which got everything running again. 

As far as I can tell only beancount-import has an sklearn dependency in my environment, but perhaps some other package installed the latest version prior to beancount import. Thought I'd raise this in case others run into the same issue.

**Comment by v3c70r** [comment]
I also had this issue, but it seems it has been fixed in PR #180.

**Comment by Zburatorul** [comment]
Yes, OP please make sure you the have the right version of scikit and b-i.
I suggest fresh environment, clone master, and then `pip install -e .` in the folder.
